package.json

```diff
@@ -52,7 +52,7 @@
     "tap": "^16.0.1"
   },
   "peerDependencies": {
-    "newrelic": ">=6.11.0"
+    "newrelic": ">=8.15.0"
```

Comment:
Whats the driver for bumping this to 8.15?

Reply:
meh, saw maurice was doing it. ill just revert and when we ship the agent we can bump to 9.x

Reply:
ok. we often don't bump cause it is only important when the API changes.

Reply:
oh ok. but don't we want 9.x because we'll drop support for older node versions?

Reply:
it won't really matter cause that will be handled by the engines field. you'll notice many of these are on 6.x. but up to ya'll.

Reply:
Bob had suggested on my AWS SDK module PR to bump the peer dependency. If we're not doing this, that's fine. (I'm admittedly not familiar with the automation the team has in place.)

Reply:
that was different for that PR because you needed a minimum version to remove that code you had done